patches/minecraft/net/minecraft/core/dispenser/ShearsDispenseItemBehavior.java.patch

```diff
@@ -0,0 +1,25 @@
+--- a/net/minecraft/core/dispenser/ShearsDispenseItemBehavior.java
++++ b/net/minecraft/core/dispenser/ShearsDispenseItemBehavior.java
+@@ -26,7 +_,7 @@
+       Level level = p_123580_.m_7727_();
+       if (!level.m_5776_()) {
+          BlockPos blockpos = p_123580_.m_7961_().m_121945_(p_123580_.m_6414_().m_61143_(DispenserBlock.f_52659_));
+-         this.m_123573_(m_123576_((ServerLevel)level, blockpos) || m_123582_((ServerLevel)level, blockpos));
++         this.m_123573_(m_123576_((ServerLevel)level, blockpos) || tryShearLivingEntity((ServerLevel)level, blockpos, p_123581_));
+          if (this.m_123570_() && p_123581_.m_220157_(1, level.m_213780_(), (ServerPlayer)null)) {
+             p_123581_.m_41764_(0);
+          }
+@@ -53,11 +_,11 @@
+       return false;
+    }
+ 
+-   private static boolean m_123582_(ServerLevel p_123583_, BlockPos p_123584_) {
```

Comment:
I believe there are problems with removing private methods, as someone could have ATed that method. Probably better to keep the old private one around and have it pass in `ItemStack.EMPTY` or `new ItemStack(Items.SHEARS)`

Reply:
Yeah - don't patch out methods unless explicitly necessary.  What you can do is add the new method right below the old one, and change the body of the old one to invoke the new one with `ItemStack.EMPTY` as the stack param.  This keeps the method in place and prevents the patch from looking really weird as well.